server/tests/output/formatter/jimi_test.py

```diff
@@ -517,7 +517,7 @@ def test_placeline_washington(self):
     def test_format_content(self):
         item = self.format_item(
             {
-                "body_html": "<p>Body HTML<br>test remove bold<b> </b> and <i>idiom</i></p>"
+                "body_html": "<p>Body HTML<br>test remove bold <b>  </b>and <b>bold1</b>and <i>idiom</i>and removed empty bold <b> </b></p>"
```

Comment:
can you add space before `and` so it's like `<b> </b> and` ?

Reply:
yes